community/onboarding-offboarding.md

```diff
@@ -5,50 +5,51 @@ toc=true
 # Onboarding/Offboarding
 
 ## How to onboard someone in KOSS?
-
 1. Add them to the “Newbies” GitHub team.
 1. Add them to the kossiitkgp@ Google Group. Check the delivery settings. Anyone can ask to join using this link https://groups.google.com/g/kossiitkgp
-1. Add them to the Slack.
+1. Add them to the Slack. Ask each one to install the Slack desktop and mobile apps and enable notifications to not miss out any important updates.
 1. Add them to their batch’s private channel.
-1. Add their contact details on https://github.com/kossiitkgp/secrets .
+1. Add their contact details on https://github.com/kossiitkgp/secrets.
 1. Assign their 1:1 group.
 1. Add their name to the website.
-1. Add them to facebook groups (unofficial)
 1. Give access to KOSS’s facebook page.
-1. Make them store numbers of everyone in team.
-1. Add them to [KOSS Workshops](https://teams.microsoft.com/l/team/19%3aa012c782fe2d428ab5bf644ef564efe5%40thread.tacv2/conversations?groupId=5fa16cd8-ce71-4c62-b41c-22816c93f841&tenantId=71dbb522-5704-4537-9f25-6ad2dcd4278d) Microsoft Team and make them Owner (Applicable for online workshop conduction)
+1. Make them store numbers of everyone in team using the automatically generated VCFs in the secrets repository.
+1. Ask them to create their batch's private Whatsapp group and add it to the "Kossverse" community. 
+1. Create a Whatsapp group with the Executives + CTMs and name it `KOSS XX` where `XX` is the sum of the last two digits of the batch years of the executives and CTMs. Eg: Batch of '21 and batch of '22 will have a whatsapp group named `KOSS 43`. (Credits to [@rakaar](https://github.com/rakaar) for the naming scheme).
+1. Create a Whatsapp group with all the advisors (B.Tech and Dual), Executives, and CTMs, and name it `KOSS XX` where `XX` is the sum of the last two digits of their batch years.
```

Comment:
All in-campus advisors should be definitely present. How about advisors (last 2 batches), execs and ctms?

Reply:
Are non-incampus people called advisors?

Reply:
Yes. All the alumni are advisors. But adding everyone seems a bit too much no?
That's why I was suggesting the latest 2 batches of advisors. In that case some members of the oldest batch will be alums but that's fine ig. Give them an option to either join grp or not :)

Reply:
I assumed only on campus people are called advisors, so assumed it is the last two batches. [This](https://github.com/kossiitkgp/docs/blob/master/community/governance.md#hierarchy) section supports my assumption.

Reply:
I'll still add the on campus clause

Reply:
done